src/plugins/OauthPlugin/OauthPlugin.js

```diff
@@ -6,28 +8,43 @@ import { DialogPlugin } from '@/plugins/DialogPlugin';
 import { MessengerGroupPlugin } from '@/plugins/MessengerGroupPlugin';
 import OauthApi from '@/plugins/OauthPlugin/OauthPublicApi';
 import { DIRECTION, PLUGIN_METHODS, PLUGIN_NAMES } from '@/constants';
-import oauthHandlers from './oauthHandlers';
+import oauthHandlers from '@/plugins/OauthPlugin/oauthHandlers';
 import FrameStrategy from '@/plugins/OauthPlugin/FrameStrategy';
 
 const documentsCheckReg = /\/documents$/gi;
 
 export default class OauthPlugin extends PluginBase {
+  /**
+   * @returns {string}
+   */
   static get pluginName() {
     return PLUGIN_NAMES.OAUTH;
   }
 
+  /**
+   * @returns {OauthHandlers}
```

Comment:
do we need type in d.ts? maybe just declare in this place only?

Reply:
@dudiq is there exist any reason to mix types definition (especially with jsdoc notations) and clean logic? No. All declarations should be in the declaration files. All logic should be in the *.js files. This convention makes code clean and shiny, without trash types in every file; and we'll always knew where exactly lives type declaration - in *.d.ts file.

When we declare stand-alone type, we build a strong base for the future and remove possible WET at fetal stage. And much more, when types placed in one folder - in one file! look at the OauthPlugin.d.ts - it's pretty useful see all of them on the screen, without annoying requirement switch between files.

Mix types and logic definitely isn't good.